releases/release-1.30/README.md

```diff
@@ -0,0 +1,121 @@
+| **KubeCon EU**                                                                         |                               | Tuesday 19th - Friday 22nd March 2024                                                                               | week 11  |                                                        |
+| **Begin [Burndown]** (Monday, Wednesday, and Friday meetings)                          | Lead                          | Monday 18th March 2024                                                                                             | week 11  |                                                        |
+| **[Test Freeze]**                                                                      | Branch Manager                | [01:00 UTC Wednesday 20th March 2024 / 18:00 PDT Tuesday 19th March 2024](https://everytimezone.com/s/e016958f) | week 11  |                                                        |
+| release-1.30 branch created                                                            | Branch Manager                | Tuesday 19th March 2024                                                                                            | week 11  |                                                        |
+| release-1.30 jobs created                                                              | Branch Manager                | Tuesday 19th March 2024                                                                                            | week 11  |                                                        |
+| Major Themes deadline                                                                  | Comms                         | Tuesday 19th March 2024                                                                                             | week 11  |                                                        |
+| Start final draft of Release Notes                                                     | Release Notes Lead            | Tuesday 19th March 2024                                                                                            | week 11  |                                                        |
+| Release blog ready to review                                                           | Comms / Docs                  | [01:00 UTC Wednesday 20th March 2024 / 18:00 PDT Tuesday 19th March 2024](https://everytimezone.com/s/e016958f) | week 11  |                                                        |
+| 1.30.0-beta.0 released                                                                 | Branch Manager                | Thursday 21st March 2024                                                                                           | week 11  |                                                        |
```

Comment:
Maybe we could have beta.0 on March 5 or March 6 (closer to the beginning of the code freeze)